resources/assets/js/app/models/VoteModel.js

```diff
@@ -1,6 +1,27 @@
+var Backbone = require('backbone');
 var BaseModel = require('../instances/Model');
 var _ = require('underscore');
+var moment = require('moment');
+var DateHelper = require('../helpers/dateHelper.js');
 
 module.exports = BaseModel.extend({
-    urlRoot: '/votes'
+    urlRoot: '/votes',
+    minTime: 5,
+    validate: function (attrs) {
+        var errors = {};
+        if (!attrs.title || attrs.title.trim().length == 0)
+            errors['title'] = 'Write question title!';
```

Comment:
Please replace title string ''Write question title!'' to 'Title field is required'